mesh_map/src/mesh_map.cpp

```diff
@@ -411,12 +463,18 @@ bool MeshMap::initLayerPlugins()
     layer_plugin->updateLethal(lethals, empty);
     if (!layer_plugin->readLayer())
     {
+      RCLCPP_INFO_STREAM(node->get_logger(), "Computing layer '" << layer_name << "' ...");
       layer_plugin->computeLayer();
+      
+      // RCLCPP_INFO_STREAM(node->get_logger(), "Writing '" << layer_name << "' to file.");
+      // layer_plugin->writeLayer();
+      // RCLCPP_INFO_STREAM(node->get_logger(), "Finished writing '" << layer_name << "' to file.");
```

Comment:
Please delete these lines, if they are not necessary anymore.

Or, if you think they might become useful later, maybe make them `DEBUG` logs. This will ensure they get considered in refactorings and won't become weird dead code (comments) in the long run.

Reply:
done